test/test_rentlinx.rb

```diff
@@ -1,8 +1,23 @@
 require 'minitest/autorun'
 require 'rentlinx'
+require 'vcr'
 
 # Test for Rentlinx module
-class TestRentlinx < MiniTest::Unit::TestCase
+class TestRentlinx < MiniTest::Test
+  def setup
+    # We use hide the real values using VCR
+    @auth_key = ENV['RENTLINX_AUTH_KEY'] || '<AUTH_KEY>'
+    @password = ENV['RENTLINX_PASSWORD'] || '<PASSWORD>'
+    @username = ENV['RENTLINX_USERNAME'] || '<USERNAME>'
+    @site_url = ENV['RENTLINX_SITE_URL'] || 'http://localhost'
```

Comment:
To your other point, I really wanted to avoid needing to add a configuration file just for the tests, plus additional logic to load the file in the tests. I see your argument for not having to add more stuff to our environment, but I also think that having a configuration file with private credentials that lives anywhere in the repo directories, even if gitignored, may eventually result in a potential leakage of credentials.

I'm certainly open to making this as easy as possible in the future in combination with considering security concerns. For now, I just wanted to have something simple in place that addresses that security side of the tests. Great feedback.